packages/panzer/dof-mgr/src/Panzer_BlockedDOFManager.hpp

```diff
@@ -150,11 +150,16 @@ class BlockedDOFManager : public UniqueGlobalIndexer<LocalOrdinalT,std::pair<int
    /** \brief Get the global IDs for a particular element. This function
      * overwrites the <code>gids</code> variable.
      */
-   void getElementGIDs(LocalOrdinalT localElmtId,std::vector<GlobalOrdinal> & gids,const std::string & blockIdHint="") const; // ?
+   void getElementGIDs(panzer::LocalOrdinal localElmtId,std::vector<GlobalOrdinal> & gids,const std::string & blockIdHint="") const; // ?
+
+   /** \brief Get the global IDs for a particular element. This function
+     * overwrites the <code>gids</code> variable.
+     */
+  void getElementGIDsPair(panzer::LocalOrdinal localElmtId,std::vector<std::pair<int,GlobalOrdinal>> & gids,const std::string & blockIdHint="") const;
```

Comment:
Could you add a line to the documentation on this function that explains what is in the pair (int vs global ordinal).